### PR TITLE
Update docs to talk about using new, puppet managed merge day instance.

### PR DESCRIPTION
**Pull request by mozbhearsum**
Mihai, I think you said you're probably going to be doing the merges? I'd really love it if you could try using the new instance for them, to make sure everything works.

### CODE REVIEW

File: docs/mergeduty/merge-and-staging-instance.md [hunk 1]
```diff
@@ -1,44 +1,5 @@
...
-# ssh = ssh -C -i ~/.ssh/sfraser_rsa.pub
-ssh = ssh -C -i ~/.ssh/jlund_rsa.pub
-
-[extensions]
-strip =
-$ df -h  # check disk space. You will need at least 25gb on /
-$ rm -rf ~/merge_day_57.0 # if needed, you can clear previous cycle migrations that were completed, like so
-$ mkdir ~/merge_day_${RELEASE_VERSION_FOR_CYCLE} # create a new work dir. This will be used for all migrations in the current cycle
-```
+To access it, start the "mergeday1" instance in us-west-2 from the AWS console. You should be able to access it by sshing to mergeday1.srv.releng.usw2.mozilla.com. Be sure to "sudo su - buildduty" before doing any work.
```
Comment (MihaiTabara) [inline docs/mergeduty/merge-and-staging-instance.md]: Quick question: do we need to specifically forward the ssh agent when we login to make sure the migration scripts won't barf?

Comment (mozbhearsum) [reply docs/mergeduty/merge-and-staging-instance.md]: Probably, yes. Going over these docs I think there's a lot of room for improvement. I was focusing on the "replace ad-hoc instance with mergeday1" part, though -- I'm not knowledgable enough about merge day stuff in general to feel confident writing in detail about it.

**Comment by mozbhearsum** [comment]
Do you want to leave this PR open and unmerged until we do a succesful merge with the new instance?

File: docs/mergeduty/merge-and-staging-instance.md (hunk 1 above)
Comment (MihaiTabara) [reply docs/mergeduty/merge-and-staging-instance.md]: No worries, I'll gladly do that on the fly soon enough :)

**Comment by MihaiTabara** [comment]
Neah, let's merge. Change is good, let's embrace it. I'll update the docs on the fly on that Monday. If I get stuck, I still have the old instance to rely on + I can glance at the closed PRs. Let's not block this. 

Me and @lundjordan are meeting briefly next week to discuss the steps anyway so we should refine what we know then.

**Pull request merged**